website/docs/r/glue_security_configuration.html.markdown

```diff
@@ -0,0 +1,76 @@
+---
+layout: "aws"
+page_title: "AWS: aws_glue_security_configuration"
+sidebar_current: "docs-aws-resource-glue-security-configuration"
+description: |-
+  Manages a Glue Security Configuration
+---
+
+# aws_glue_security_configuration
+
+Manages a Glue Security Configuration.
+
+## Example Usage
+
+```hcl
+resource "aws_glue_security_configuration" "example" {
+  name = "example"
+
+  encryption_configuration {
+    cloudwatch_encryption {
+      cloudwatch_encryption_mode = "DISABLED"
+    }
+
+    job_bookmarks_encryption {
+      job_bookmarks_encryption_mode = "DISABLED"
+    }
+
+    s3_encryption {
+      kms_key_arn        = "${data.aws_kms_key.example.arn}"
+      s3_encryption_mode = "SSE-KMS"
+    }
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `encryption_configuration` – (Required) Configuration block containing encryption configuration. Detailed below.
+* `name` – (Required) Name of the security configuration.
+
+### encryption_configuration Argument Reference
+
+* `cloudwatch_encryption` - (Required) Configuration block containing encryption configuration for CloudWatch. Detailed below.
+* `job_bookmarks_encryption` - (Required) Configuration block containing encryption configuration for job bookmarks. Detailed below.
+* `s3_encryption` - (Required) Configuration block containing encryption configuration for S3 data. Detailed below.
+
+#### cloudwatch_encryption Argument Reference
```

Comment:
IMO this should be:

```suggestion
---

A `cloudwatch_encryption` block as defined below:
```

Reply:
Hmm 🤔 We don't use horizontal rule style anywhere else. Probably a standard we should discuss out of band and potentially fix everywhere at once.